### PR TITLE
add reactions and change naming of few compounds

### PR DESCRIPTION
**Pull request by sleepbot-zapp**

### CODE REVIEW

File: reactions.json [hunk 1]
```diff
@@ -163,12 +165,22 @@
     "HgSO4 / H2SO4": "propanone"
   },
   "methanal": {
-    "HCN": "formaldehyde cyanohydrin",
-    "NaHSO3": "formaldehyde sodium bisulphite"
+    "HCN": "hydroxyethanenitrile",
+    "NaHSO3": "hydroxymethanesulfonic acid",
```
Comment (CodeWithSwastik) [inline reactions.json]: Where is the sodium?

File: reactions.json [hunk 2]
```diff
@@ -163,12 +165,22 @@
...
+    "Red P / HBr": "methane",
+    "KMnO4": "methanoic acid",
+    "K2Cr2O7 / H2SO4": "methanoic acid",
+    "LiAlH4": "methanol"
   },
   "ethanal": {
-    "HCN": "acetaldehyde cyanohydrin",
-    "NaHSO3": "acetaldehyde sodium bisulphite"
+    "HCN": "hydroxypropionitrile",
+    "NaHSO3": "hydroxyethanesulfonic acid",
```
Comment (CodeWithSwastik) [inline reactions.json]: Where is the sodium?

File: reactions.json [hunk 3]
```diff
@@ -208,9 +224,13 @@
     "KI": "iodobenzene",
     "HBF4 / (Heat)": "fluorobenzene"
   },
+  "nitrobenzene":{
+    "Zn / HCl": "Aniline",
```
Comment (CodeWithSwastik) [inline reactions.json]: Names should be Lowercase

File: reactions.json [hunk 4]
```diff
@@ -190,14 +202,18 @@
     "Br2 / CCl4": "4-bromophenol",
     "conc. HNO3": "2,4,6-trinitrophenol",
     "CO2 / NaOH / H+ (423K, 3-7 atm)": "2-hydroxybenzoic acid",
-    "H2SO4/ (373K)": "4-hydroxysulphonic acid"
+    "H2SO4/ (373K)": "4-hydroxysulphonic acid",
+    "NaOH / CO2 / H+": "2-hydroxybenzoic acid",
```
Comment (CodeWithSwastik) [inline reactions.json]: This is the same as the reaction in line 204

File: reactions.json [hunk 5]
```diff
@@ -190,14 +202,18 @@
     "Br2 / CCl4": "4-bromophenol",
     "conc. HNO3": "2,4,6-trinitrophenol",
     "CO2 / NaOH / H+ (423K, 3-7 atm)": "2-hydroxybenzoic acid",
-    "H2SO4/ (373K)": "4-hydroxysulphonic acid"
+    "H2SO4/ (373K)": "4-hydroxysulphonic acid",
+    "NaOH / CO2 / H+": "2-hydroxybenzoic acid",
+    "CHCl3 / aq KOH": "2-hydroxybenzaldehyde"
```
Comment (CodeWithSwastik) [inline reactions.json]: There should be a . after aq

**Review by CodeWithSwastik — CHANGES_REQUESTED** [review]

File: reactions.json (hunk 4 above)
Comment (sleepbot-zapp) [reply reactions.json]: unusre about the name

File: reactions.json (hunk 3 above)
Comment (sleepbot-zapp) [reply reactions.json]: fixed

File: reactions.json (hunk 1 above)
Comment (sleepbot-zapp) [reply reactions.json]: name confoosion

File: reactions.json (hunk 2 above)
Comment (sleepbot-zapp) [reply reactions.json]: name confoosion

File: reactions.json [hunk 6]
```diff
@@ -163,12 +166,22 @@
     "HgSO4 / H2SO4": "propanone"
   },
   "methanal": {
-    "HCN": "formaldehyde cyanohydrin",
-    "NaHSO3": "formaldehyde sodium bisulphite"
+    "HCN": "hydroxyethanenitrile",
+    "_NaHSO3": "hydroxymethanesulfonic acid",
```
Comment (CodeWithSwastik) [inline reactions.json]: Why is there an underscore? also the name is still not correct

Comment (sleepbot-zapp) [reply reactions.json]: the underscore is there because I ain't sure about the name

**Review by CodeWithSwastik — APPROVED** [review]

**Comment by sleepbot-zapp** [comment]
ye

**Pull request merged**